javascript/javascript1/week1/goodBoyOldBoy.js

```diff
@@ -0,0 +1,19 @@
+
+    var dogYearOfBirth = 1980;
+    var dogYearFuture = 2022;
+    var dogYear = dogYearFuture - dogYearOfBirth;
+    /* 
+    0 for dog years
+    1 for human years */
+    var shouldShowResultInDogYears = 1; 
```

Comment:
you can make `shouldShowResultInDogYears` boolean instead of a number like this
```js
const shouldShowResultInDogYears = true;
```
Then you can do the condition like this
```js
if (shouldShowResultInDogYears) {
}
else {
}
```